### DISCUSSION TITLE
kanban-python: Kanban Terminal App written in Python (setup with pyscaffold)

### DISCUSSION THREAD

**Discussion by Zaloog** [pyscaffold/pyscaffold#740] [Show and tell]
Pyscaffold was a great help for developing / publishing  my first package.
I learned alot working on that project and still have a lot to learn.
I have not used the docs builder yet, rather made an comprehensive README.md to demonstrate functionalities.

You can have a look at it here: [github repo](https://github.com/Zaloog/kanban-python)

Thank you very much for creating/managing Pyscaffold @abravalheri and @FlorianWilhelm.

**Comment by abravalheri** [comment]
Thank you very much @Zaloog !!!
